src/main/java/com/google/sps/servlets/ContestsServlet.java

```diff
@@ -66,22 +67,41 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     Election election = Election.fromEntity(electionEntityOptional.get());
 
     List<JsonElement> contestJsonList =
-        ImmutableSet.copyOf(election.getContests())
+        election
+            .getContests()
             .stream()
             .map(id -> KeyFactory.createKey(Contest.ENTITY_KIND, id.longValue()))
             .map(key -> ServletUtils.getFromDatastore(datastore, key))
-            .map(entity -> Contest.fromEntity(entity).toJsonString(datastore))
-            .map(jsonString -> JsonParser.parseString(jsonString))
-            .collect(ImmutableList.toImmutableList());
+            .map(
+                entity -> {
+                  if (entity.isPresent()) {
+                    return JsonParser.parseString(
+                        Contest.fromEntity(entity.get()).toJsonString(datastore));
+                  } else {
+                    return JsonNull.INSTANCE;
+                  }
+                })
```

Comment:
small nit: can you change this to a ternary operator?
```
entity -> entity.isPresent() 
  ? JsonParser.parseString(
                        Contest.fromEntity(entity.get()).toJsonString(datastore))
  : JsonNull.Instance
```

Reply:
Yup -- let me make that change right now,